packages/redux/src/returns/selectors.ts

```diff
@@ -68,8 +69,10 @@ export const getReturnError = (state: StoreState) => getError(state.returns);
  *
  * @returns {object} Return item.
  */
-export const getReturnItem = (state: StoreState, returnItemId: string) =>
-  getEntity(state, 'returnItems', returnItemId);
+export const getReturnItem = (
+  state: StoreState,
+  returnItemId: string,
+): ReturnItem => getEntity(state, 'returnItems', returnItemId);
```

Comment:
My previous comment is reflected here - why don't we use `getEntityById` instead?

Reply:
Answered on previous comment. Checking with my team :)